presto-hive/src/main/java/com/facebook/presto/hive/security/ranger/VXUser.java

```diff
@@ -0,0 +1,180 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.facebook.presto.hive.security.ranger;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import javax.annotation.concurrent.Immutable;
+
+import java.util.List;
+
+import static com.google.common.base.MoreObjects.toStringHelper;
+
+@Immutable
+public class VXUser
+{
+    private String createDate;
+    private String description;
+    private String firstName;
+    private List<Object> groupIdList;
+    private List<String> groupNameList;
+    private Long id;
+    private Long isVisible;
+    private String lastName;
+    private String name;
+    private String password;
+    private Long status;
+    private String updateDate;
+    private List<String> userRoleList;
+    private Long userSource;
+
+    @JsonCreator
+    public VXUser(
+            @JsonProperty("createDate") String createDate,
+            @JsonProperty("description") String description,
+            @JsonProperty("firstName") String firstName,
+            @JsonProperty("groupIdList") List<Object> groupIdList,
+            @JsonProperty("groupNameList") List<String> groupNameList,
+            @JsonProperty("id") Long id,
+            @JsonProperty("isVisible") Long isVisible,
+            @JsonProperty("lastName") String lastName,
+            @JsonProperty("name") String name,
+            @JsonProperty("password") String password,
+            @JsonProperty("status") Long status,
+            @JsonProperty("updateDate") String updateDate,
+            @JsonProperty("userRoleList") List<String> userRoleList,
+            @JsonProperty("userSource") Long userSource)
+    {
+        this.createDate = createDate;
```

Comment:
This will always be populated by REST api response and never manually set for null checking. 
How about we restrict the attributes to the limited members used in the plugin and remove others?